Dockerfile

```diff
@@ -1,22 +1,19 @@
 FROM alpine:3.4
 MAINTAINER Zalando SE
 
-RUN apk --no-cache add ca-certificates curl
+RUN apk --no-cache upgrade && apk --no-cache add ca-certificates
 
 # add Zalando CA bundle
-RUN curl https://secure-static.ztat.net/ca/zalando-root.ca > /usr/local/share/ca-certificates/zalando-root.crt
-RUN curl https://secure-static.ztat.net/ca/zalando-service.ca > /usr/local/share/ca-certificates/zalando-service.crt
+ADD https://secure-static.ztat.net/ca/zalando-root.ca /usr/local/share/ca-certificates/zalando-root.crt
+ADD https://secure-static.ztat.net/ca/zalando-service.ca /usr/local/share/ca-certificates/zalando-service.crt
 
 # add AWS RDS CA bundle
-RUN mkdir /tmp/rds-ca && \
-    curl https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem > /tmp/rds-ca/aws-rds-ca-bundle.pem
+ADD https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem /tmp/rds-ca/aws-rds-ca-bundle.pem
```

Comment:
In my opinion it is better to use curl as the docker best practices at https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy state that ADD should only be used to for local tar file extraction and for all other either COPY or some other tool like curl should be used. Even if this means that curl needs to be installed and removed afterwards for assembling the image.